### DISCUSSION TITLE
Propose for different tiers on open collective

### DISCUSSION THREAD

**Discussion by prisis** [faker-js/faker#985] [Ideas]
The current tiers are leftovers from the old faker.js open collective page. 

I would like to propose different tiers for open collective.

Backers -> 5$ / month
Support us with a monthly donation and help us continue our activities.
Your name will be listed in README and BACKERS files.

Bronze Sponsor -> $50 / month
You are serious about supporting OSS!
Your name will be listed under a dedicated "Generous Backers" section in README and BACKERS files.
Note: if $50 is too much, you can also choose any custom amount between $5 and $50 to qualify as a normal backer. (dont know if this works on opencollective)

Silver Sponsor -> $100 a month
Your name or company logo (small) will be put in README and BACKERS files.

Gold Sponsor -> $250 or 500 a month
You can have your company's logo placed on:
The front page of faker, the README and BACKERS files with a link

Platinum Sponsor -> $500 or 1000 a month
A custom sponsorship message and location. (Contact us to discuss the details)
The README and BACKERS files
The possibility to set a priority for a PR / issue

You will still have the possibility to spend custom amount of money.

What do you think about this change

**Comment by ejcheng** [comment]
Looks good! Could you please be more specific about "the possiblity to set a priority for a PR / issue"? Do you mean one issue per month, their input being considered on all PRs/issues, etc? Thanks

**Reply by ST-DDT** [reply]
I think this looks good too.

> Could you please be more specific about "the possiblity to set a priority for a PR / issue"?

I don' think this is necessary as these kind of interactions are usually quite rare.
If they are paying good money for it, then I'm willing to give them some leeway with their number of wishes.
(Note, setting a priority is not equal to us doing things instantly, we just give their needs priority like releasing a fix sooner)

**Reply by ejcheng** [reply]
> I think this looks good too.
> 
> > Could you please be more specific about "the possiblity to set a priority for a PR / issue"?
> 
> I don' think this is necessary as these kind of interactions are usually quite rare. If they are paying good money for it, then I'm willing to give them some leeway with their number of wishes. (Note, setting a priority is not equal to us doing things instantly, we just give their needs priority like releasing a fix sooner)

Makes sense, thanks for the clarification!

**Comment by prisis** [comment]
New proposal

Backers -> 2$ / month
Support us with a monthly donation and help us continue our activities.
Note: Getting the new fancy faker stickers (soon)

Bronze Sponsor -> $100 / month
Become a bronze sponsor with a monthly donation of $100 and get your logo on BACKERS.md on Github. 
You will receive an invoice after the payment.
Note: t-shirt (soon)

Silver Sponsor -> $250 a month
Become a silver sponsor with a monthly donation of $250 and get your logo on BACKERS.md on Github and the sponsor page on http://fakerjs.dev/. (You will receive an invoice after the payment.)

Gold Sponsor -> 500 a month
Become a gold sponsor with a monthly donation of $500 and get your logo on our README.md on Github and the frontpage of http://fakerjs.dev/. You will receive an invoice after the payment.
The possibility to set a priority for a PR / issue

Highest Backers will be displayed on every doc page (with the new vitepress sponsors feature)

**Comment by RichardLitt** [comment]
Side note: Let me know if there is anything that I or @open-source-Collective can do to help out with your use of the platform, too. Happy to listen and see if there is anything I can do to make your use of Open Collective better.
